nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/src/main/java/org/apache/nifi/event/transport/netty/NettyEventServerFactory.java

```diff
@@ -156,9 +168,13 @@ private void setChannelOptions(final AbstractBootstrap<?, ?> bootstrap) {
             bootstrap.option(ChannelOption.SO_RCVBUF, socketReceiveBuffer);
             bootstrap.option(ChannelOption.RCVBUF_ALLOCATOR, new FixedRecvByteBufAllocator(socketReceiveBuffer));
         }
+
         if (socketKeepAlive != null) {
             bootstrap.option(ChannelOption.SO_KEEPALIVE, socketKeepAlive);
         }
+
+        // ChannelOption only takes integer for time in milliseconds? strange
```

Comment:
This comment seems somewhat superfluous, although interesting, it is probably with the goal of limiting the theoretically maximum timeout.